module/config/argument/argument.yaml

```diff
@@ -586,7 +586,8 @@ SupplyPack:
 Minigame:
   Collect: false
 PrivateQuarters:
-    BuyRoses: true
+    Filter: |-
+      GiftRoses
```

Comment:
Maybe just keep `BuyRoses: bool` for easier understanding. I think there won't be more gifts in the future, no need to have such high customization and extensibility.